src/ray/protobuf/common.proto

```diff
@@ -827,18 +827,23 @@ enum TaskStatus {
   // to the worker. It will be executed shortly. For actor tasks, execution may be
   // delayed to satisfy ordering constraints or argument fetching.
   SUBMITTED_TO_WORKER = 5;
+  // The actor task is fetching arguments. This happens after the actor worker
+  // receives the task and the actor task has object refs as arguments.
+  PENDING_ACTOR_TASK_ARGS_FETCH = 6;
+  // The actor task is waiting due to ordering or concurrency constraints.
+  PENDING_ACTOR_TASK_ORDERING_OR_CONCURRENCY = 7;
```

Comment:
Wait, why are these specific to actor tasks @jjyao?

Reply:
For normal task, args fetch is before SUBMITTED_TO_WORKER and happens in raylet and is `PENDING_ARGS_FETCH`. Also normal task, you don't need to wait for concurrency, each a worker process only runs one normal task at a time.

Reply:
Understood. It's a bit odd to encode the entity name into the state, but i see your point there.